### PR TITLE
Translated to English

### PR DESCRIPTION
**Pull request by 87johnboyle**
App translated best I can see.

### CODE REVIEW

File: src/Components/AddRide.js [hunk 1]
```diff
@@ -31,27 +31,27 @@ class AddRide extends Component{
...
             return false;
         }
         if(this.state.title.length>500){
-            this.setState({popup:'Описот несмее да биде над 500 букви '+this.state.title.length+'/500'})
+            this.setState({popup:'Description must not be more than 500 characters '+this.state.title.length+'/500'})
             return false;
         }
         if(isNaN(this.state.hour)){
-            this.setState({popup:'Ве молиме внесете валиден број во полето за часови'})
+            this.setState({popup:'Enter a valid number in the hour field'})
```
Comment (johnpooley) [inline src/Components/AddRide.js]: I feel like 'please enter' would be better. I might create a 'politeness' branch later on :)

File: src/Components/Map.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ class Map extends Component{
         this.map = new mapboxgl.Map({
         container: this.mapContainer,
         style: 'mapbox://styles/mapbox/streets-v11',
-        center: [21.43141,41.99646],
+        center: [55.8596,-4.2503],
```
Comment (johnpooley) [inline src/Components/Map.js:44]: I take it this is changing the location to Glasgow as well?

**Review by johnpooley — APPROVED** [review]

File: src/Components/Map.js (hunk 2 above)
Comment (87johnboyle) [reply src/Components/Map.js:44]: Ah yes, apologies. Forgot I'd put that in. Zoom level isn't sorted yet.

File: src/Components/AddRide.js (hunk 1 above)
Comment (87johnboyle) [reply src/Components/AddRide.js]: Yeah good point, probably a good idea to have a text style guide we can work to. Tried to keep it consistent, but for a first past there are bound to be inconsistencies.

Updated the commit.

Comment (johnpooley) [reply src/Components/AddRide.js]: its maybe something that we can decide on once we can see the actual site from a user perspective

**Review by johnpooley — APPROVED** [review]

**Pull request merged**